### PR TITLE
Tizen .NET Hot Reload feature update.

### PR DESCRIPTION
**Pull request by om90prakash**
### Change Description ###

Describe your changes here.

For instance,
 - Fix broken links of native application docs.
 -  ...

### Bugs Fixed ###

Provide links to bugs here

For instance,

 - Issue #265

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR: ACR-1120

### CODE REVIEW

**Review by sangwook96 — APPROVED** [review]
It's so excellent and Thank for making a good guide.

File: docs/application/vstools/tools/dotnet-hotreload.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+No matter how you use .NET Hot Reload please be aware that some changes are not supported at runtime and will prompt you with a rude edit dialog and require you to restart your app in order to apply. [Supported Edits in Tizen .NET Hot Reload](https://github.com/dotnet/roslyn/blob/main/docs/wiki/EnC-Supported-Edits.md)
+
+
+# Examples
+
+## Tizen .NET Hot Reload without breakpoint
+
+This example shows using Hot Reload in an Tizen ElmSharp based UI application.
+Here, we are adding a callback method on button click event and once application is launched on target device and running in debug mode, we will modify the
+callback method and do Ctrl+S (Save) to apply Hot Reload.
```
Comment (omar-srbd) [inline docs/application/vstools/tools/dotnet-hotreload.md:35]:
```suggestion
Here, we add a callback method on the button click event and once the application is launched on a target device and it is running in debug mode, we modify the callback method and Save (Ctrl+S) to apply Hot Reload.
```

File: docs/application/vstools/tools/dotnet-hotreload.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+The following video demonstrates the same on Tizen Device:
```
Comment (omar-srbd) [inline docs/application/vstools/tools/dotnet-hotreload.md:37]:
```suggestion
The following video demonstrates the example on a Tizen device:
```

File: docs/application/vstools/tools/dotnet-hotreload.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+[![Hotreload Example Video](media/hotreload_without_bp.png)](media/Hotreload_Without_Breakpoint.mp4 "Hotreload Example Video Without Breakpoint")
```
Comment (omar-srbd) [inline docs/application/vstools/tools/dotnet-hotreload.md:39]:
```suggestion
[![Hotreload Example Video](media/hotreload_without_bp.png)](media/Hotreload_Without_Breakpoint.mp4 "Hot Reload example video without breakpoint"){:target="_blank"}
```

File: docs/application/vstools/tools/dotnet-hotreload.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+
+## Tizen .NET Hot Reload with breakpoint
```
Comment (omar-srbd) [inline docs/application/vstools/tools/dotnet-hotreload.md:42]:
```suggestion
### Tizen .NET Hot Reload with breakpoint
```

File: docs/application/vstools/tools/dotnet-hotreload.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+This example shows using Hot Reload in an Tizen ElmSharp based UI application.
```
Comment (omar-srbd) [inline docs/application/vstools/tools/dotnet-hotreload.md:44]:
```suggestion
The example shown here uses Hot Reload in a Tizen ElmSharp based UI application.
```

File: docs/application/vstools/tools/dotnet-hotreload.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+Here, we are adding a breakpoint in the main method on a static method call and once application is launched on target device and running in debug mode, we will modify the method and do ctrl+S (Save) to apply Hot Reload.
+
+The following video demonstrates the same on Tizen Device:
```
Comment (omar-srbd) [inline docs/application/vstools/tools/dotnet-hotreload.md:47]:
```suggestion
The following video demonstrates the example on a Tizen device:
```

File: docs/application/vstools/tools/dotnet-hotreload.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+[![Hotreload Example Video](media/hotreload_with_bp.png)](media/Hotreload_With_Breakpoint.mp4 "Hotreload Example Video With Breakpoint")
```
Comment (omar-srbd) [inline docs/application/vstools/tools/dotnet-hotreload.md:49]:
```suggestion
[![Hotreload Example Video](media/hotreload_with_bp.png)](media/Hotreload_With_Breakpoint.mp4 "Hot Reload Example video with breakpoint"){:target="_blank"}
```

**Review by omar-srbd — COMMENTED** [review]
Hello @om90prakash, I appreciate you merging both documents.
I have reviewed the file and left some correction suggestions. Please check and incorporate them at your earliest convenience.
Thank you.

**Review by omar-srbd — COMMENTED** [review]
@om90prakash, please check the suggestions.
Also, https://github.com/Samsung/tizen-docs/pull/1791#discussion_r1141623246 from the first review needs to be addressed.
Thank you.

File: docs/application/vstools/tools/dotnet-hotreload.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
+# Tizen .NET Hot Reload
+
+With Hot Reload you can now modify your apps managed source code while the application is being debugged, without the need to stop and restart the debugging.
+After starting the debugging of the application on the emulator or real device, you can apply the changed file without stopping the debugging of the application.
+With Hot Reload changes are applied only on the second entrance to the function. If the change you made is supported your app will now be patched while its running with your new logic and you should see the changes in your app’s behavior the next time the updated code is re-executed by either your action or by something like a timer triggering the code.
```
Comment (omar-srbd) [inline docs/application/vstools/tools/dotnet-hotreload.md:5]:
```suggestion
With Hot Reload changes are applied only on the second entrance to the function. If the change you made is supported, your app will now be patched while it's running with your new logic. You should see the changes in your app’s behavior the next time the updated code is re-executed by either your action or by a timer triggering the code.
```

**Review by omar-srbd — COMMENTED** [review]
@om90prakash, please check the suggestion.
Thank you.

File: docs/application/vstools/tools/dotnet-hotreload.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+**2.** Go to **Debugging &gt; .NET / C++ Hot Reload** and check all the checkboxes.
+
+**Figure: VS options page to enable .NET Hot Reload**
+
+![Tizen options page to enable .NET Hot Reload](media/enable_vs_hotreload.png)
+
+
+## Supported/Unsupported changes and architecture
+Currently, Tizen .NET Hot Reload is supported on aarch64 and armv7l.
+No matter how you use .NET Hot Reload please be aware that some changes are not supported at runtime and will prompt you with a rude edit dialog and require you to restart your app in order to apply the changes. [Supported Edits in Tizen .NET Hot Reload](https://github.com/dotnet/roslyn/blob/main/docs/wiki/EnC-Supported-Edits.md)
```
Comment (omar-srbd) [inline docs/application/vstools/tools/dotnet-hotreload.md:26]: External links in a .md file open in a new window
```suggestion
No matter how you use .NET Hot Reload please be aware that some changes are not supported at runtime and will prompt you with a rude edit dialog and require you to restart your app in order to apply the changes. [Supported Edits in Tizen .NET Hot Reload](https://github.com/dotnet/roslyn/blob/main/docs/wiki/EnC-Supported-Edits.md){:target="_blank"}
```

**Review by omar-srbd — COMMENTED** [review]
Hello @om90prakash, please incorporate the final change in this PR.
Thank you.

File: docs/application/vstools/tools/dotnet-hotreload.md (hunk 9 above)
Comment (sangwook96) [reply docs/application/vstools/tools/dotnet-hotreload.md:26]: @omar-srbd Hi, could you review it again ?

Comment (omar-srbd) [reply docs/application/vstools/tools/dotnet-hotreload.md:26]: @om90prakash, as per the commits history in this PR, I can see that this specific change was not made in previous commits.
The instance where the change was made and had to be reverted was in the video links of **lines 39 & 49**.
The change requested here is for an external link redirecting users to a different site and hence should open in a new tab.

**Review by omar-srbd — APPROVED** [review]
@om90prakash, thanks for incorporating all the changes as per the suggestions.

@asm-foysal, I am approving this PR for merging.

**Pull request merged**